controllers/dnsupdate.go

```diff
@@ -52,7 +53,7 @@ func (r *GslbReconciler) getGslbIngressIPs(gslb *k8gbv1beta1.Gslb) ([]string, er
 			gslbIngressIPs = append(gslbIngressIPs, ip.IP)
 		}
 		if len(ip.Hostname) > 0 {
-			IPs, err := Dig(r.Config.EdgeDNSServer, ip.Hostname)
+			IPs, err := utils.Dig(r.Config.EdgeDNSServer, ip.Hostname)
 			if err != nil {
 				return nil, err
```

Comment:
as `log.Info()` was removed from `Dig` -  we need to explicitly log it here, otherwise the failure of resolution will be hidden from the end user

Reply:
@ytsarev  thx for finding 👍 . I forgot .  Will log in caller function(s) as dnsupdate keeps dependency to log.